source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py

```diff
@@ -182,6 +182,41 @@ def body_incoming_wrench(env: ManagerBasedEnv, asset_cfg: SceneEntityCfg) -> tor
     return link_incoming_forces.view(env.num_envs, -1)
 
 
+def grab_images(
+    env: ManagerBasedEnv,
+    sensor_cfg: SceneEntityCfg = SceneEntityCfg("tiled_camera"),
+    data_type: str = "rgb",
+    convert_perspective_to_orthogonal: bool = True,
```

Comment:
Is there a reason this is `True` by default?
Since this is mostly used in RL, the policy will probably handle both cases like a champ.
In that case I would opt for the route with the lowest compute by default.

Reply:
I originally set this to `True` by default because I was thinking about using point cloud observation space, in which case, `unproject_depth` requires an "orthogonal" depth image.

However, since this method doesn't do any point cloud stuff, only depth image, I agree that the RL policy should easily adapt to perspective depth, and that it's better to have a lower compute default. Good spot!

TLDR: changing default to `False`